### PR TITLE
Add Google CLA requirements to PR

### PR DESCRIPTION
**Pull request by dima-avdeev-jb**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -11,3 +11,8 @@ Test: Describe how you tested your changes. Note that this line (with `Test:`) i
 ## Issues Fixed
 
 Fixes: [Optional] The bug on [https://issuetracker.google.com](https://issuetracker.google.com) being fixed
+
+## Google CLA
```
Comment (MatkovIvan) [inline .github/PULL_REQUEST_TEMPLATE.md:15]: Let's try CLA bot check instead https://github.com/apps/google-cla

Comment (pjBooms) [reply .github/PULL_REQUEST_TEMPLATE.md:15]: I think, we need to add a note about CLA anyway.  We just don't need a screenshot to be attached

Comment (dima-avdeev-jb) [reply .github/PULL_REQUEST_TEMPLATE.md:15]: @pjBooms Good suggestion!

Comment (dima-avdeev-jb) [reply .github/PULL_REQUEST_TEMPLATE.md:15]: Removed screenshot information. And left information about Google CLA.

**Review by pjBooms — APPROVED** [review]

**Pull request merged**